### PR TITLE
Pablo vahanian

### PR DESCRIPTION
**Pull request by pvahanian**

### CODE REVIEW

**Review by pvahanian — COMMENTED** [review]
This is my final version.

File: about.html [hunk 1]
```diff
@@ -12,53 +12,87 @@
...
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+    <img src="img/jumbo-about.png" alt="skyline" class="jumboskyline">
 
+    <div class="containerabout-page">
+   
+   <h2>About</h2>
 
+    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
```
Comment (agyin3) [inline about.html:33]: Just a reminder to always have text wrapped in a text element `p`, `h1`, `h2`, `h3`, etc.

File: index.html [hunk 2]
```diff
@@ -13,15 +13,26 @@
 
 <body>
     <div class="container">
```
Comment (agyin3) [inline index.html:15]: Great job with marking up and styling your header. Just to keep your page more semantic I'd use a `<header>` tag in place of the `<div>`

**Review by agyin3 — APPROVED** [review]
Overall with your styling look at ways to have your code follow DRY(Don't Repeat Yourself) principles

**Pull request merged**